src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const arrOfPopulation = [];
+
+population.forEach((element) => {
+  arrOfPopulation.push(Number(element.innerHTML.split(',').join('')));
```

Comment:
Ensure that the `element.innerHTML` contains valid numeric values. If any element contains non-numeric data or is empty, `Number()` will return `NaN`, which could affect the total and average calculations. Consider adding a check to handle such cases.